src/Exporter.php

```diff
@@ -120,7 +120,7 @@ public function shortenedExport($value)
             return sprintf(
                 '%s Object (%s)',
                 get_class($value),
-                count($this->toArray($value)) > 0 ? '...' : ''
+                method_exists($value, '__toString') ? (string)$value : (count($this->toArray($value)) > 0 ? '...' : '')
```

Comment:
Sorry. Didn't notice the PR author. Wouldn't have commented if I'd seen it was you. :)